ui.apps/src/main/content/jcr_root/apps/hclecomm/config.author.qa/org.apache.sling.commons.log.LogManager.factory.config-hclecomm.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="sling:OsgiConfig"
+    org.apache.sling.commons.log.file="logs/project-hclecomm.log"
```

Comment:
@DeepaliAgrwl : why is this config required ?

Reply:
deleted...